data/net/minecraft/core/BlockPos.mapping

```diff
@@ -94,12 +103,24 @@ CLASS net/minecraft/core/BlockPos
 		ARG 5 maxX
 		ARG 6 maxY
 		ARG 7 maxZ
+	METHOD randomInCube (Ljava/util/Random;ILnet/minecraft/core/BlockPos;I)Ljava/lang/Iterable;
+		COMMENT Creates an Iterable with n BlockPos Objects in a cube around the center with the specified maxDistance.
```

Comment:
Once again, the comment doesn't make sense, no need to specify `BlockPos Objects` and I would argue that saying it makes an iterable is pointless.

Honestly what is here is basically just the `@return` javadoc imo.